src/store/actions/actions-model.ts

```diff
@@ -375,3 +375,7 @@ export const setLastDirectorChangeDate: ActionIF = ({ commit }, date: string): v
 export const setWarnings : ActionIF = ({ commit }, businessWarnings: Array<BusinessWarningIF>): void => {
   commit('mutateBusinessWarnings', businessWarnings)
 }
+
+export const setClickFileAndPay: ActionIF = ({ commit }, clickFileAndPay: boolean): void => {
+  commit('mutateClickFileAndPay', clickFileAndPay)
+}
```

Comment:
Was there no other means already in place that is prompted when clicking file and file? 
A validation flag perhaps?

I'm surprised that this is something that isn't already handled?

Reply:
After another look, isn't `setValidateSteps` already a trigger for validations?

Reply:
I'll find out what happens if I change "setClickFileAndPay" to "setValidateSteps".